packages/gitbook/src/components/PageAside/ScrollSectionsList.tsx

```diff
@@ -30,9 +31,12 @@ export function ScrollSectionsList(props: { sections: DocumentSection[] }) {
         });
     }, [sections]);
 
+    const bodyLoaded = useBodyLoaded();
+
     const activeId = useScrollActiveId(ids, {
         rootMargin: `-${HEADER_HEIGHT_DESKTOP}px 0px -40% 0px`,
         threshold: SECTION_INTERSECTING_THRESHOLD,
+        additionalEffects: bodyLoaded,
```

Comment:
I think we should name it `active` or `enabled`

and only scrolled when it's `true`